tests/epyccel/test_epyccel_lists.py

```diff
@@ -0,0 +1,127 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring, missing-class-docstring
+# coding: utf-8
+""" Tests for list methods.
+"""
+
+import pytest
+from pyccel.epyccel import epyccel
+
+@pytest.mark.parametrize( 'language', [
+        pytest.param("c", marks = [
+            pytest.mark.skip(reason="append() not implemented in c"),
+            pytest.mark.c]),
+        pytest.param("fortran", marks = [
+            pytest.mark.skip(reason="append() not implemented in fortran"),
+            pytest.mark.fortran]),
+        pytest.param("python", marks = pytest.mark.python)
+    ]
+)
```

Comment:
Could we not define a `language` fixture in this file, rather than copying this long decorator many times?

Reply:
Is it possible to put xfail marks in a fixture? The xfail marks should be removed in a few weeks or months once the method is supported

Reply:
could you please explain how that would be possible? I was unable to do so since I'm not that familiar with writing test files using pytest so I did in the most basic way

Reply:
> Is it possible to put xfail marks in a fixture? The xfail marks should be removed in a few weeks or months once the method is supported

It should be possible:
```python
@pytest.fixture( params=[
        pytest.param("fortran", marks = [
            pytest.mark.skip(reason="list methods not implemented in fortran"),
            pytest.mark.fortran]),
        pytest.param("c", marks = [
            pytest.mark.skip(reason="list methods not implemented in c"),
            pytest.mark.c]),
        pytest.param("python", marks = pytest.mark.python)
    ],
    scope = "module"
)
def language(request):
    return request.param
```

Reply:
can you check the refactored version please

Reply:
Looks good! Have you checked that it actually works? 😉

Reply:
It did work in my linux machine 🤞

Reply:
Looks good but I wonder if we should name this fixture something like `language_python_ok`. At some point we will have some tests in this file which can use the existing `language` fixture (without the xfails) and some which need this one (and maybe some that need one where only C or only Fortran fails)

Reply:
Good point. How about using multiple test files and move the tests from one to the other as they start passing?

Reply:
This is also possible if you have a suggestion for a good name? We may need up to 4 files:
- Passing in Python only
- Passing in C and Python
- Passing in Fortran and Python
- Passing in C, Fortran and Python

Reply:
Well, using four files is a bit cumbersome... We could call them
- `test_epyccel_lists.py`
- `test_epyccel_lists__no_c.py`
- `test_epyccel_lists__no_fortran.py`
- `test_epyccel_lists__python_only.py`

Another solution is using a single file, but providing a utility function which generates the correct decorators with minimum user input.

In any case I suggest that we tackle this in a future PR.